flocx_market/objects/base.py

```diff
@@ -0,0 +1,32 @@
+from oslo_log import log
+from oslo_versionedobjects import base as object_base
+import datetime
+
+LOG = log.getLogger(__name__)
+
+
+class MarketPlaceObject(object_base.VersionedObject):
```

Comment:
I should have mentioned this earlier; let's call this `FLOCXMarketObject` to keep the naming consistent